(DOCSP-19878) December 02 2021 Release (#1507)
mongodb/docs-realm: source/services.txt

   GitHub Service Snippets </services/snippets/github>
   Send Mobile Push Notifications </services/send-mobile-push-notifications>

> .. important:: Third Party Service Deprecation

Reviewer: This note is great, but I'd probably make it an include and put it on every page in the section, in case someone gets to the page directly (via search or a link) and misses this info.

Maybe with a link to the note about re-adding them to the UI in ln 70? Or make that an include on each service page, also?

Author: Agreed. Added this note as an include to each page.

   Third party services in {+service+} have been deprecated in favor of creating 

>    HTTP endpoints using :ref:`external dependencies <external-dependencies>`.

Author: ```suggestion
   HTTP endpoints that use :ref:`external dependencies <external-dependencies>`.
```

   GitHub Service Snippets </services/snippets/github>
   Send Mobile Push Notifications </services/send-mobile-push-notifications>

> .. important:: Third Party Service Deprecation
> 
>    Third party services in {+service+} have been deprecated in favor of creating 
>    HTTP endpoints and by using third-party SDKs by with NPM packages.
>    
>    You can no longer create new third-party services, but existing services will 
>    continue to work. 
> .. for how long???

Author: ```suggestion
.. important:: Third Party Service Deprecation

   Third party services in {+service+} have been deprecated in favor of creating 
   HTTP endpoints using :ref:`external dependencies <external-dependencies>`.
   
   You can no longer create new third-party services, but existing services will 
   continue to work until December 2022.
```

   This section will remain in the documentation until such time.  

Introduction
------------
Modern applications often use multiple external services to handle complex use 
cases, such as messaging, analytics, and data management. You can connect to 

> these services through MongoDB Realm by creating and configuring service 

Author: ```suggestion
these services through {+service+} by creating and configuring service 
```
